functions/Test-DbaDbCompression.ps1

```diff
@@ -570,84 +581,69 @@ IF OBJECT_ID('tempdb..##tmpEstimatePage', 'U') IS NOT NULL
                         Continue
                     }
                     #Execute query against individual database and add to output
                     foreach ($row in ($server.Query($sql, $db.Name))) {
                         [pscustomobject]@{
-                            ComputerName                  = $server.ComputerName
-                            InstanceName                  = $server.ServiceName
-                            SqlInstance                   = $server.DomainInstanceName
-                            Database                      = $row.DBName
-                            Schema                        = $row.Schema
-                            TableName                     = $row.TableName
-                            IndexName                     = $row.IndexName
-                            Partition                     = $row.Partition
-                            IndexID                       = $row.IndexID
-                            IndexType                     = $row.IndexType
-                            PercentScan                   = $row.PercentScan
-                            PercentUpdate                 = $row.PercentUpdate
-                            RowEstimatePercentOriginal    = $row.RowEstimatePercentOriginal
-                            PageEstimatePercentOriginal   = $row.PageEstimatePercentOriginal
+                            ComputerName = $server.ComputerName
+                            InstanceName = $server.ServiceName
+                            SqlInstance  = $server.DomainInstanceName
+                            Database     = $row.DBName
+                            Schema       = $row.Schema
+                            TableName    = $row.TableName
+                            IndexName    = $row.IndexName
+                            Partition    = $row.Partition
+                            IndexID      = $row.IndexID
+                            IndexType    = $row.IndexType
+                            PercentScan  = $row.PercentScan
+                            PercentUpdate = $row.PercentUpdate
+                            RowEstimatePercentOriginal = $row.RowEstimatePercentOriginal
+                            PageEstimatePercentOriginal = $row.PageEstimatePercentOriginal
                             CompressionTypeRecommendation = $row.CompressionTypeRecommendation
-                            SizeCurrent                   = [dbasize]($row.SizeCurrentKB * 1024)
-                            SizeRequested                 = [dbasize]($row.SizeRequestedKB * 1024)
-                            PercentCompression            = $row.PercentCompression
+                            SizeCurrent  = [dbasize]($row.SizeCurrentKB * 1024)
+                            SizeRequested = [dbasize]($row.SizeRequestedKB * 1024)
+                            PercentCompression = $row.PercentCompression
```

Comment:
Just an FYI, the first time someone opens this up in VS Code and commits a change the formatting will go back the way it was 😀 🔢 ☣️ 

A ton of PRs I have seen commands have one minor change from the contributor and the remainder are formatting changes caused by the editor of choice. I don't really know how we can fix that unless we enforce rules and catch things like this.